bin/start-vm

```diff
@@ -16,13 +16,20 @@ set -Eeuo pipefail
 # Define default VARs
 gnu_head="head"
 gnu_stat="stat"
+build_os="$(uname -s)"
+arch="$(uname -m)"
```

Comment:
Maybe we should use [get_arch.sh](https://github.com/gardenlinux/gardenlinux/blob/main/get_arch.sh) for this. This script is also called in the `Makefile` and `build.sh`, so we would align `start-vm` with both other implementations.

In addition, it also normalizes the architecture, so it always returns arm64, no matter if the host system identifies itself as `arm64` or `aarch64`.

What do you think?